hacks_mbof/wsgi.py

```diff
@@ -10,7 +10,9 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
+from whitenoise.django import DjangoWhiteNoise
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "hacks_mbof.settings")
 
 application = get_wsgi_application()
+application = DjangoWhiteNoise(application)
```

Comment:
These two lines of assignment to `application` is awkward.  It would be better as one line:

```python
application = DjangoWhiteNoise(get_wsgi_application())
```

If you feel it must use two lines, use a different variable name for the first assignment, like:

```python
wsgiApplication = get_wsgi_application()
staticApplication = DjangoWhiteNoise(wsgiApplication) 
```

You'd need to change the `hacks_mbof.wsgi:application` reference in `Dockerfile` to `hacks_mbof.wsgi:staticApplication`, too.